mw-webapp/src/component/sidebar/SidebarContent/SidebarContent.tsx

```diff
@@ -0,0 +1,28 @@
+import {PropsWithChildren} from "react";
+import {
+  Close as DialogClose,
+  Content as DialogContent,
+  Overlay as DialogOverlay,
+  Portal as DialogPortal,
+} from "@radix-ui/react-dialog";
```

Comment:
@radix-ui/react-dialog does not added to dependencies in the package.json and it leads to an error even if I run "pnpm i" command